docs/interpreter/postgresql.md

```diff
@@ -44,10 +49,19 @@ Postgresql Interpreter will be deprecated and merged into JDBC Interpreter. You
     <td></td>
   </tr>
 </table>
+---
+
+## Overview
+
+[<img align="right" src="http://img.youtube.com/vi/wqXXQhJ5Uk8/0.jpg" alt="zeppelin-view" hspace="10" width="250"></img>](https://www.youtube.com/watch?v=wqXXQhJ5Uk8)
 
-----
+This interpreter seamlessly supports the following SQL data processing engines:
 
-## PostgreSQL, HAWQ  Interpreter for Apache Zeppelin
+* [PostgreSQL](http://www.postgresql.org/) - OSS, Object-relational database management system (ORDBMS)
+* [Apache HAWQ](http://pivotal.io/big-data/pivotal-hawq) - Powerful [Open Source](https://wiki.apache.org/incubator/HAWQProposal) SQL-On-Hadoop engine.
```

Comment:
probably need to update this to `Apache HAWQ (incubating)`
and http://hawq.incubator.apache.org/